frontend/src/CommitInfo.res

```diff
@@ -225,28 +221,6 @@ let make = (~repoId, ~pullNumber=?, ~worker, ~setLastCommit) => {
           }}
         </Column>
       </Row>
-      {switch status {
-      | Fail
-      | Cancel
-      | Running =>
```

Comment:
This could still be useful to show when the latest benchmark run has failed or been cancelled?

Reply:
@art-w Could you comment on this change? I think the FE warning about old commit could still be useful for a failed/cancelled run? Or am I missing something?

Reply:
Yes I have a patch in progress!

Reply:
Great, thanks!